scripts/post-processing/3-cluster_analysis.R

```diff
@@ -0,0 +1,158 @@
+# C.Savonen, CCDL for ALSF
+# 2019
+# 
+# Purpose: Analyze clustering of metadata scRNA-seq gene matrix .tsv using 
+#          kmeans and KNN clustering
+#          
+# "-d" : Path to gene matrix in tab delimited format, gene x sample with gene 
```

Comment:
What's the use case for reading all the TSV files in a directory in? It makes me a little bit nervous.

Reply:
That's how I'm using it right now, because I don't wanna have to write in all the names of the files I'm reading in. I have all my different normalization datasets in one folder and I just want them all read in. But I added the other method of explicitly writing the file names so that way if files are mixed in with other file types, one could still multiple files in.

Reply:
Could you do the "all the TSV folders in a directory" bit in the shell script -> loop through and run this multiple times?

Reply:
What's the advantage to doing this?

Reply:
The advantage is probably limited to this being reused outside of this context where someone has both metadata and gene matrix data in the same directory.

Reply:
Alrighty. I'll change it. But this script also has the capability of taking the file names explicity so it seems like someone would just do that. IDK. I'll change it though.

Reply:
This is not a dealbreaker for me as much as "a thing that makes me nervous so we should talk/think about it." So feel free to leave as is.